client/src/components/Fab.tsx

```diff
@@ -0,0 +1,35 @@
+import { useState, useEffect } from 'react';
+import arrow from '../assets/arrow.svg';
+
+function Fab() {
```

Comment:
Can you rename this to `ToTopButton`? I feel like because this button serves a specific purpose, it shouldn't be called a generic "Fab".